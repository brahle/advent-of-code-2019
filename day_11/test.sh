#!/bin/bash

DIR=$(pwd)
cd out/production/day_11

scala com.brahle.adventofcode.year2019.Test < $DIR/test/test.$1.in | grep -e "\(OUTPUT\|DONE\)" > out

diff -q -b $DIR/test/test.$1.out out
if [ $? -ne 0 ]
then
    echo "EXPECTED: "
    cat $DIR/test/test.$1.out

    echo ""
    echo "GOT: "
    cat out
    exit 1
fi
