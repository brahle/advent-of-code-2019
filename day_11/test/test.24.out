OUTPUT: 1219070632396864
DONE 1102
